internal/cmd/stack/stack.go

```diff
@@ -174,6 +174,19 @@ func Command() *cli.Command {
 					},
 				},
 			},
+			{
+				Name:  "outputs",
+				Usage: "Shows current outputs for a specific stack. Does not show sensitive outputs.",
```

Comment:
Maybe we need to tweak this to "Does not show the value of sensitive outputs", on the grounds you're displaying them but just masking the value?